src/docbuilder/rustwide_builder.rs

```diff
@@ -504,14 +506,26 @@ impl RustwideBuilder {
             .purge_from_cache(&self.workspace)
             .map_err(FailureError::compat)?;
         local_storage.close()?;
-        if let Some(distribution_id) = self.config.cloudfront_distribution_id_web.as_ref() {
+
+        for distribution_id in [
+            self.config.cloudfront_distribution_id_web.as_ref(),
+            self.config.cloudfront_distribution_id_static.as_ref(),
```

Comment:
note to myself: we invalidate the wrong paths for the `static` distribution here. 

For `static.docs.rs` we need to invalidate the `/rustdoc/krate` and `/sources/krate` prefixes, for example.